ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/poe.yml

```diff
@@ -1,14 +0,0 @@
----
-### PoE Global ###
-poe:
-  reboot:
-    action: maintain
-  interface_shutdown:
-    action: power-off
-
-lldp:
-  tlvs:
-    - name: power-via-mdi
-      transmit: false
```

Comment:
looks like this one did not make it to lldp -> is it because it is not needed for coverage (transmit true/false)?

Reply:
yes transmit true/false is already covered hence not adding it again.
https://github.com/aristanetworks/avd/pull/4722/files#diff-0fb9733021cf92ba8e50fddf3d54c2b3d272d2bdf9e64237cded4ae7f4d20490R11